README.md

```diff
@@ -0,0 +1,29 @@
+# gp-v2-services
+
+This repository contains backend code for [Gnosis Protocol V2](TODO high level gpv2 link) written in Rust.
+
+## Order Book
+
+The `orderbook` crate provides the http api through which users (usually through a frontend web application) interact with the order book. Users can add signed orders to the order book and query the state of their orders.
+
+Solvers also interact with the order book by querying a list of open orders that they can attempt to settle.
+
+The api is documented with [openapi](https://protocol-rinkeby.dev.gnosisdev.com/api/).
+
+The order book service itself uses PostgreSQL as a backend to persist orders. In addition to connecting the http api to the database it also checks order validity based on the on chain time, trade events, erc20 funding and approval so that solvers can query only valid orders.
```

Comment:
```suggestion
The order book service itself uses PostgreSQL as a backend to persist orders.
In addition to connecting the http api to the database it also checks order validity based on the block time, trade events, erc20 funding and approval so that solvers can query only valid orders.
```